### ISSUE TITLE
sqlutil: to implement cop and op "trunc" operator

### ISSUE THREAD

**Issue by pvanek** [qorelanguage/qore#2363]
at least for oracle and postgresql.
No idea about mysql and sybase/mssql...

**Comment by pvanek** [comment]
cop is already implemented.
https://docs.qore.org/current/modules/SqlUtil/html/group__sql__cop__funcs.html#ga823e286e413a11dd21d4665e93704822

we just need op_... version AFAIK

**Comment by davidnich** [comment]
@pvanek yes we need an `op_` version - I tried it with `cop_` in the where clause, and it didn't work
